src/shared/particle_dynamics/diffusion_reaction_dynamics/diffusion_dynamics.h

```diff
@@ -151,7 +151,7 @@ class CorrectedKernelGradientContact
  * @brief Base class for diffusion relaxation process between two contact bodies.
  */
 template <class ParticlesType, class ContactParticlesType, class KernelGradientType = KernelGradientContact>
-class BaseDiffusionRelaxationContact
+class BaseDiffusionRelaxationBoundary
```

Comment:
still keep as contact because diffusion between contact bodies would be derived from here also.